src/steamship/cli/deploy.py

```diff
@@ -0,0 +1,218 @@
+import importlib
+import os
+import traceback
+import zipfile
+from abc import ABC, abstractmethod
+from pathlib import Path
+
+import click
+from semver import VersionInfo
+
+from steamship import Package, PackageVersion, Steamship, SteamshipError
+from steamship.cli.manifest_init_wizard import validate_handle, validate_version_handle
+from steamship.cli.ship_spinner import ship_spinner
+from steamship.data.user import User
+from steamship.invocable.lambda_handler import get_class_from_module
+from steamship.invocable.manifest import Manifest
+
+DEFAULT_BUILD_IGNORE = [
+    "build",
+    ".git",
+    ".venv",
+    ".ipynb_checkpoints",
+    ".DS_Store",
+    "venv",
+    "tests",
+    "examples",
+    ".idea",
+    "__pycache__",
+]
+
+
+def update_config_template(manifest: Manifest):
+
```

Comment:
did you run the pre-commits?

Reply:
... yes?  Do you see something you think should have been flagged?